aequilibrae/project/network/network.py

```diff
@@ -239,6 +246,437 @@ def create_from_osm(
 
         logger.info("Network built successfully")
 
+    def create_from_gmns(
+        self,
+        link_file_path: str,
+        node_file_path: str,
+        use_group_path: str = None,
+        geometry_path: str = None,
+        srid: int = 4326,
+    ) -> None:
+        """
+        Creates AequilibraE model from links and nodes in GMNS format.
+
+        Args:
+            *link_file_path* (:obj:`str`): Path to a links csv file in GMNS format
+
+            *node_file_path* (:obj:`str`): Path to a nodes csv file in GMNS format
+
+            *use_group_path* (:obj:`str`): Optional argument. Path to a csv table containing groupings of uses. This helps AequilibraE
+            know when a GMNS use is actually a group of other GMNS uses
+
+            *geometry_path* (:obj:`str`): Optional argument. Path to a csv file containing geometry information for a line object, if not
+            specified in the link table
+        """
+
+        p = Parameters()
+        gmns_l_fields = p.parameters["network"]["gmns"]["link_fields"]
+        gmns_n_fields = p.parameters["network"]["gmns"]["node_fields"]
+
+        # Collecting GMNS fields names
+
+        gmns_link_id_field = gmns_l_fields["link_id"]
```

Comment:
can probably do in a single line? Not all that important....

Reply:
Isso agora está no arquivo gmns_builder.py. Mudei um pouco disso de forma que o que era importante virou um atributo da classe GMNSBuilder().